tests/test_resources/test_clusters/test_kubernetes_cluster.py

```diff
@@ -0,0 +1,31 @@
+import tests.test_resources.test_clusters.test_cluster
+
+import pytest
+
+import runhouse as rh 
+
+def num_cpus():
+    import multiprocessing
+    return f"Num cpus: {multiprocessing.cpu_count()}"
+
+
+@pytest.mark.kubernetestest
+class TestKubernetesCluster(tests.test_resources.test_clusters.test_cluster.TestCluster):
+
+    MAP_FIXTURES = {"resource": "cluster"}
+
+    UNIT = {"cluster": []}
+    LOCAL = {"cluster": []}
+    MINIMAL = {"cluster": ["kubernetes_cpu_cluster"]}
+    THOROUGH = {"cluster": []}
```

Comment:
let's include kubernetes_cpu_cluster in thorough and maximal. setting it to [] overrides it and excludes it from those